doc/guide_B2C_tenant.md

```diff
@@ -0,0 +1,188 @@
+
+Add the Energy Track & Trace URL `https://energytrackandtrace.dk/da/callback` as redirect URL. Click `Configure`.
+
+![New app registration](images/App_registration_frontend_redirect_urls_new_props.png)
+
+Add a second redirect URL `https://energytrackandtrace.dk/en/callback` to the list, and click `Save`.
+
+![New app registration](images/App_registration_frontend_redirect_urls_new_2nd.png)
+
+Add API permission to allow user sign-in for this specific app. Navigate to `App registration` and `API permissions`.
+
+Add permissions to app registration.
+
+![Frontend app registration permissions](images/App_registration_frontend_permissions_add.gif)
+
+Grant admin consent to permissions.
+
+![Frontend app registration permissions](images/App_registration_frontend_permissions_admin_consent.png)
+
+### Deploy custom policies
+
+#### MitID Client Secret
+
+Client secret for MitID integration must be securily stored as a secret in Azure. The secret is referenced by the MitID custom policy deployed later in this guide.
+
+Navigate to `Identity Experience Framework`.
+
+![Identity Experience Framework](images/Custom_policy_secret_identity_experience_framework.png)
+
+Navigate to `Policy keys`.
+
+![Policy keys](images/Custom_policy_keys.png)
+
+Add a new secret value. The value must match the secret configured with the MitID provider. Click `Add`. Choose `Manual`, enter name `B2C_1A_MitIDSecret` and provide the secret value in the Secret input field. Key usage must be set to `Signature`.
+
+![Policy keys](images/Custom_policy_mitid_secret.png)
+
+#### Token Signing Secret
+
+Add secrets used for signing tokens. In the `Policy keys` view, click `Add.`. Choose `Generate`, fill in name `B2C_1A_TokenEncryptionKeyContainer` and choose Key Type `RSA`. Key usage must be set to `Signature`.
+
+![Policy keys](images/Custom_policy_token_signing_key.png)
+
+#### Token Encryption Secret
+
+Perform the same steps as in [link text](#token-signing-secret). Name should be set to `B2C_1A_TokenEncryptionKeyContainer` instead.
+
+![Policy keys](images/Custom_policy_token_encryption_key.png)
+
+#### Policies
+
+The following custom policies must be uploaded to B2C.
+
+- domains/authorization/custom_policies/production/B2C_1A_MitId.XML
+- domains/authorization/custom_policies/production/B2C_1A_ClientCredentials.XML
+
+First step is to replace GUIDs and URLs in the custom profiles. The relevant GUIDs and URLs to be replaced are marked with XML comments in the files.
+
+![Policy keys](images/Custom_policy_upload.png)
+
+![Policy keys](images/Custom_policy_upload_mitid.png)
+
+![Policy keys](images/Custom_policy_upload_client_credentials.png)
+
+Notice that it's not possible to test the custom policy from the Azure portal without added a web redirect in the frontnend app registration.
+
+Add a web application to the ett frontend app with redirect URL `https://jwt.ms`. Not the login can be tested directly from the Azure portal.
```

Comment:
Should we include this? jwt.ms can only show "code" returned, that can be used to get a bearer token. So we can only see the login was successfull, but not mapping of claims was right.

Reply:
I am not sure why to use jwt.ms? Is it to test the policy?

Reply:
yes

Reply:
Can add it to a test section?